README.md

```diff
@@ -47,6 +47,20 @@ Import `Ecto.SoftDelete.Schema` into your Schema module, then add `soft_delete_s
   end
 ```
 
+If you want to make sure auto-filtering is disabled for a schema, set the `auto_exclude_from_queries?` option to false
+
+```elixir
+  defmodule User do
+    use Ecto.Schema
+    import Ecto.SoftDelete.Schema
+
+    schema "users" do
+      field :email,           :string
```

Comment:
nit: formattings a bit off here

Reply:
I noticed that too, but it's like that in the other examples. We can do a doc cleanup later rather than putting it on @caioaao's plate

Reply:
I noticed it too, but since everything was like that I thought it was a choice. I'll fix it